dhis-2/dhis-web/dhis-web-dataentry/src/main/webapp/dhis-web-dataentry/javascript/form.js

```diff
@@ -1889,15 +1889,14 @@ function insertDataValues( json )
     else{
     	
     	var orgUnitClosed = false;
-    	
     	$.each( organisationUnitList, function( idx, item )
-        {    		
-    		orgUnitClosed = dhis2.de.validateOrgUnitOpening( organisationUnits[item.uid], period ) ;
+        {
+            orgUnitClosed = dhis2.de.validateOrgUnitOpening( item, period ) ;
     		
     		if( orgUnitClosed )
-    		{    			  	        
-    			return;
-    		}            
+			{
+				return false;
```

Comment:
jQuery requires that you return false to break an each loop (see https://api.jquery.com/jquery.each/):
> We can break the $.each() loop at a particular iteration by making the callback function return false. Returning non-false is the same as a continue statement in a for loop; it will skip immediately to the next iteration.

(I can't seem to match the original styling whatever I do, which is why it's picking up styling changes as well)